conceptual/Npgsql/connection-string-parameters.md

```diff
@@ -78,6 +79,13 @@ Target Session Attributes | Determines the preferred PostgreSQL target server ty
 Load Balance Hosts        | Enables balancing between multiple hosts by round-robin.                   | false
 Host Recheck Seconds      | Controls for how long the host's cached state will be considered as valid. | 10
 
+## Multiplexing
+
+Parameter                                      | Description                                                                                                  | Default
+---------------------------------------------- | -----------------------------------------------------------------------------------------------------------  | -------------------------
+Multiplexing                                   | Enables multiplexing, which allows more efficient use of connections. Introduced in 5.0.                     | false
```

Comment:
I'm still hesitant to introduce this without a serious documentation page which explains exactly how to use it. I'm worried users would turn it on and use it incorrectly. For example, in multiplexing it's very important to consume the resultset ASAP so as to release the connection to other consumers, otherwise you're blocking the entire application; to make this really usable, I think we should do resultset buffering by default when using multiplexing.

In other words, is it OK if we defer documenting these parameters until more proper general documentation is available?

Reply:
Sure, I'll remove this part.